comment/views/comments.py

```diff
@@ -105,3 +144,29 @@ def post(self, request, *args, **kwargs):
         self.comment.delete()
         context = self.get_context_data()
         return render(request, 'comment/comments/base.html', context)
+
+
+class ConfirmComment(View):
+    def get(self, *args, **kwargs):
+        key = kwargs.get('key', None)
+        try:
+            tmp_comment_dict = signing.loads(str(key))
+            model_name = tmp_comment_dict.pop('model_name')
+            model_id = tmp_comment_dict.pop('model_id')
+            app_name = tmp_comment_dict.pop('app_name')
+            tmp_comment_dict.update(
+                {
+                    'content_object': get_model_obj(app_name, model_name, model_id)
+                }
+            )
+            tmp_comment = Comment(**tmp_comment_dict)
+        except (ValueError, KeyError, signing.BadSignature):
+            raise Http404
+        if Comment.objects.comment_exists(tmp_comment):
```

Comment:
is there way to make the confirmation link valid for one time only?

Reply:
Yes the line `if Comment.objects.comment_exists(tmp_comment):` does the same thing as you ask.

Reply:
I know, I asked bcz I wanted to get rid of this line and make sure that the link is one time valid. However, I can live with this check.

Reply:
Any other solution would mean we would have to use the database or caches etc which would make things more complex. So, till we find a more efficient way, we will live have to live with this :-p